shacl12-shacl-shacl/index.html

```diff
@@ -0,0 +1,415 @@
+			pre.example-sparql:before {
+				border: 1px solid #ddd;
+			}
+
+			pre.example-sparql:before {
+				color: #888;
+				content: "Example SPARQL";
+				width: 13em;
+			}
+
+			.example-results {
+				background: #edb;
+			}
+
+			.example-results,
+			.example-results:before,
+			.example-results th,
+			.example-results td {
+				border: 1px solid #cca;
+			}
+
+			/* our syntax menu for switching */
+			div.syntaxmenu {
+				border: 1px dotted black;
+				padding: 0.5em;
+				margin: 1em;
+			}
+
+			@media print {
+				div.syntaxmenu {
+					display: none;
+				}
+			}
+		</style>
+	</head>
+
+	<body>
+		<section id="abstract">
+			<p>This document describes the Shapes Constraint Language (SHACL) validator for SHACL data, referred to as "SHACL-SHACL".</p>
+			<p>
+				This specification is published by the <a href="https://www.w3.org/groups/wg/data-shapes/">Data Shapes Working Group</a>.
+			</p>
+		</section>
+
+		<section id="sotd"></section>
+
+		<section id="introduction">
+			<h2>Introduction</h2>
+			<p class="todo">TODO</p>
+		</section>
+
+		<section id="shapes">
+			<h2>Shapes</h2>
+			<p class="todo">TODO: This section will describe the Shapes created for SHACL-SHACL.</p>
+			<p>
+				These shapes are available in the RDF resource at <a href="http://www.w3.org/ns/shacl-shacl">http://www.w3.org/ns/shacl-shacl</a>.
+			</p>
+		</section>
+
+		<section id="governance">
+			<h2>Governance</h2>
+			<p class="todo">TODO: This section will describe the goverannce arrangements in place for the continued maintenance of this Note.</p>
+		</section>
+
+		<section id="security" class="appendix informative">
+			<h2>Security and Privacy Considerations</h2>
+			<p>
+				This Shapes graph does not import into or alter the data it is evaluating so will not affect an
+				organisation's information if applied to it, only report on its SAHCL Core validity. It will also not
+				automatically export results anywhere beyond a SHACL processor's standard output channel.
+			</p>
+			<p>
+				For these reasons, this content is expected to present minimal security and privacy concerns.
+			</p>
+		</section>
+
+		<section id="validator" class="appendix normative">
+			<h2>Validator</h2>
+			<p>
+				The shapes graph of SHACL-SHACL is available as an RDF resource at <a href="http://www.w3.org/ns/shacl-shacl">http://www.w3.org/ns/shacl-shacl</a>.
+			</p>
+		</section>
+
+		<section id="ack" class="appendix informative">
+			<h2>Acknowledgements</h2>
+			<p>Many people contributed to this document, including members of the RDF Data Shapes Working Group.</p>
+		</section>
+
+		<section class="appendix informative" id="changes-12">
+			<h2>Changes between SHACL 1.0 Core and SHACL 1.2 Core</h2>
+			<p>
+				SHACL 1.0 included the total information about SHACL-SHACL inline, within an Appendix, consisting of an
+				introductory paragraph and the SHACL-SHACL RDF data only. SHACL 1.2 has dedicated this entire Note to
+				define and describe SHACL-SHACL.
```

Comment:
s/define and describe SHACL-SHACL/define and describe SHACL-SHACL and its maintenance process/g

Or, "governance" instead of "maintenance process"?

Reply:
As a W3C NOTE, it will have only informative power, so `governance` seems inaccurate, and even `maintenance process` may be an overreach.

Reply:
Thank you for the context, @TallTed .

Seeing the file rendered with @nicholascar 's link in the initial PR post, I'm now fine not putting this change into the change log.